### PR TITLE
Test cleanup

### PR DESCRIPTION
**Pull request by sbesson**
This PR does three things:

- adds `logback.xml` to `src/test/resources` to remove DEBUG-level logging
- remove the `utests` namespace from `src/test/java/ome/codecs`
- sets up `pom.xml` to have multiple execution with different dependencies excluded rather than a single one and use regexp rather than `testng.xml`

Without this PR, the Travis build should have verbose logging and only run 14 tests (the ones defined in `testng.xml`). With this PR, there should be two surefire executions, one with 14 tests and one with 2 tests (for the missing services) and the logging verbosity should be reduced.
With this

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -189,10 +189,39 @@
...
+            <goals>
+              <goal>test</goal>
+            </goals>
+            <configuration>
+              <skip>false</skip>
+              <includes>
+                <include>%regex[.*Missing.*Test.*]</include>
+              </includes>
+              <classpathDependencyExcludes>
+                 <classpathDependencyExcludes>org.openmicroscopy:ome-jai</classpathDependencyExcludes>
```
Comment (rleigh-codelibre) [inline pom.xml]: Do you need an exclude for the LuraWave classes as well?  Won't be commonly available, but should we cater for it?

**Comment by rleigh-codelibre** [comment]
This all appears fine, green and good to merge.  Just one comment regarding LuraWave testing.

File: pom.xml (hunk 1 above)
Comment (sbesson) [inline pom.xml]: You mean excluding our own stubs as well. That would makes sense for completeness, although the tests do not seem to require it interestingly.

Comment (sbesson) [inline pom.xml]: a751281 should include the additional dependency exclusion for the missing tests

**Comment by rleigh-codelibre** [comment]
Thanks.  This all looks fine to me, and is green.

**Pull request merged**